indra/newview/lloutfitslist.cpp

```diff
@@ -1509,8 +1515,16 @@ LLOutfitListGearMenu::~LLOutfitListGearMenu()
 void LLOutfitListGearMenu::onUpdateItemsVisibility()
 {
     if (!mMenu) return;
+    mMenu->setItemVisible("expand", true);
+    mMenu->setItemVisible("collapse", true);
     mMenu->setItemVisible("thumbnail", getSelectedOutfitID().notNull());
     mMenu->setItemVisible("favorite", getSelectedOutfitID().notNull());
+    mMenu->setItemVisible("inventory_settings", true);
```

Comment:
Again the question where all this is suddenly coming from when it got moved elsewhere in maint-c?